_data/header.yml

```diff
@@ -17,10 +17,3 @@ primary:
 secondary:
   # this is a key into _data/navigation.yml
   links: secondary
-
-  # to enable search, change this to an object with 'action' and
-  # 'query' keys, e.g.
-  #
-  search:
-    action: /search/
-    query: q
```

Comment:
Not sure if there is reason to keep the search `action` and `query` variables. I opted to remove them.

Reply:
Agreed, we shouldn't need these any longer.